entries/d/dan.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Dan.com": {
+    "domain": "dan.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://news.dan.com/214543-",
+    "categories": [
+      "domains"
+    ]
```

Comment:
Their [Terms of Use](https://dan.com/terms_of_use#us_export_laws) says:
> Users shall not use the Services found at this Site to collect, store or transmit any technical information or data that is controlled under U.S. Export Laws. Users shall not export or re-export, or allow the export or re-export of, the Services found at this Site in violation of any U.S. Export Laws. None of the Services found at this Site may be downloaded or otherwise exported or re-exported (i) into (or to a national or resident of) any country with which the United States has embargoed trade; or (ii) to anyone on the U.S. Treasury Department's list of Specially Designated Nationals or the U.S. Commerce Department's Denied Persons List, or any other denied parties lists under U.S. Export Laws. By using this Site and the Services found at this Site, you agree to the foregoing and represent and warrant that you are not a national or resident of, located in, or under the control of, any restricted country; and you are not on any denied parties list; and you agree to comply with all U.S. Export Laws (including “anti-boycott”, “deemed export” and “deemed re-export” regulations).

```suggestion
    ],
    "regions": [
      "-cu",
      "-ir",
      "-kp",
      "-ru",
      "-sy"
    ]
```

Reply:
Thanks for the heads up! I hadn't even noticed this! 😢